src/table/table.component.tsx

```diff
@@ -28,24 +36,37 @@ export interface TableProps {
   onPageChange: (page: number) => void;
   sort: { [column: string]: Order };
   onSort: (column: string, order: Order | null) => void;
-  onClose: (column: string) => void;
 }
 
 const Table = React.memo((props: TableProps): React.ReactElement => {
   const {
     data,
-    displayedColumns,
+    availableColumns,
     totalDataCount,
     loadedData,
     loadedCount,
     resultsPerPage,
     onPageChange,
     sort,
     onSort,
-    onClose,
   } = props;
 
+  /*
+   ** A note about the columns used in this file:
+   ** availableColumns - these are the columns passed in from RecordTable.
+   **                    They represent all columns that can currently be added to the
+   **                    display based on data received from the backend
+   ** checkedColumns   - these are the columns the user has selected to appear in the table
```

Comment:
Minor point, but I dislike the `checkedColumns` name as I think it's too based on the WIP column checkboxes selector, rather than what it will look like in the future. Maybe `selectedColumns` makes more sense? I'm not sure I care enough for you to change it though 😅

Reply:
I also prefer selectedColumns. I can change in a later PR since I'm working on a new feature right now